### PR TITLE
Misc refactoring to be GitHub ready

### PR DESCRIPTION
**Pull request by quangkhoa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
   "repository": {
     "type": "git",
-    "url": "git@src.temando.io:developer-experience/temando-open-api-ui.git"
+    "url": "git@github.com:temando/open-api-ui.git"
   },
   "keywords": [
     "ui"
   ],
-  "author": "Florie Guibert",
+  "author": "Temando developers",
```
Comment (brendo) [inline package.json]: Let's list the contributors for now, https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Comment (quangkhoa) [reply package.json]: fixed

File: package.json [hunk 2]
```diff
@@ -12,12 +12,25 @@
...
+    {
+      "name": "Brendan Abbot",
+      "email": "brendan.abbott@temando.com"
+    },
+    {
+      "name": "Khoa Tran",
+      "email": "khoa.tran@temando.com"
+    }
+  ],
   "license": "ISC",
```
Comment (brendo) [inline package.json]: LICENCE is MIT, but this states ISC. I'm not fussed either way, but just needs to be consistent

Comment (quangkhoa) [reply package.json]: updated to use `MIT`

**Pull request merged**